pegasus/sites.v3/code.org/public/css/dance-landing/dance-sponsor-logo.css

```diff
@@ -2,15 +2,27 @@
   display: flex;
   justify-content: center;
   font-size: 24px;
-  margin-top: 30px;
+  margin-top: 60px;
   clear: both;
   padding-top: 30px;
+  text-align: center;
 }
 
 .sponsor-subtext-description {
   font-size: 14px;
   text-align: center;
   padding-top: 20px;
-  margin-left: 5%;
-  margin-right: 5%;
+  margin-bottom: 85px;
+  margin-left: 18%;
+  margin-right: 18%;
+}
+
+@media screen and (min-width: 0px) and (max-width: 512px) {
```

Comment:
I don't think you need to have a separate css file just for the logo classes. I'd put this inside the dance.css file which should have all the styles needed for /dance.

Reply:
done!